cmd/keyunwrap/keyunwrap.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/square/keysync/backup"
+	"golang.org/x/crypto/nacl/box"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var b64 = base64.StdEncoding
+
+func main() {
+	var (
+		app            = kingpin.New("keyunwrap", "A tool to unwrap a keysync backup key")
+		privateKeyFile = app.Flag("privatekeyfile", "The offline private key").Required().String()
+		generate       = app.Flag("generate", "Generate a new key pair").Bool()
```

Comment:
feels like `generate` should be a subcommand of `keyunwrap` not a flag.

Reply:
Yeah, that makes this code a lot nicer too.